docs/source/geopmsession.1.rst

```diff
@@ -50,60 +45,73 @@ Get Help or Version
-signals requested though standard input is made and the results are
-printed to the screen.
+Unless the ``-n`` option is specified, a descriptive header is written
+first and then the output from reading the requests.  Unless ``-p`` is
+specified, only one line of output after the header is printed.
 
 Options
 -------
 
 -h, --help  .. _help option:
 
-    Print help message and exit
+    Print help message and exit.
 
 -v, --version  .. _version option:
 
-    Print version and exit
+    Print version and exit.
 
 -t TIME, --time TIME  .. _time option:
 
-    Total run time of the session to be opened in seconds
+    Total run time of the session to be opened in seconds.
 
 -p PERIOD, --period PERIOD  .. _period option:
 
     When used with a read mode session reads all values out periodically with
-    the specified period in seconds
+    the specified period in seconds.
 
 --pid PID  .. _pid option:
 
-    Stop the session when the given process PID ends
+    Stop the session when the given process PID ends.
+
+--print-header  .. _printheader option:
+
+    Deprecated. Now this option is the default, see --no-header.
 
---omit-header  .. _header option:
+-n, --no-header  .. _header option:
 
-    Do not print the CSV header before printing any sampled values
+    Do not print the CSV header before printing any sampled values.
 
---delimiter .. _delimiter DELIMITER .. option:
+-d, --delimiter  .. _delimiter DELIMITER .. option:
 
-    Delimiter used to separate values in CSV output
+    Delimiter used to separate values in CSV output. Default: ",".
 
---report-out .. _reportout REPORT_OUT option:
+-r, --report-out  .. _reportout REPORT_OUT option:
 
-    Output summary statistics into a yaml file. Note if ``--report-out=-``
-    is specified, the report will output to stdout.
+    Output summary statistics into a yaml file. Note if ``--report-out=-`` is
+    specified, the report will output to stdout. When used with the
+    ``--enable-mpi`` option, reports from all hosts will be combined using the
+    ``---`` document separator, and the output is written (stdout or to file) solely
+    by the MPI process "rank 0".
 
---trace-out .. _traceout TRACE_OUT option:
+-o, --trace-out  .. _traceout TRACE_OUT option:
 
     Output trace data into a CSV file. Note if ``--trace_out=-`` is specified,
-    the trace will output to stdout. To avoid gathering trace data,
-    set this parameter to ``/dev/null``
+    the trace will output to stdout which is also the default behavior. To avoid
+    gathering trace data, set this parameter to ``/dev/null``.  When used with
+    the ``--enable-mpi`` option, trace file names will be appended with the
+    hostname combined with the ``-`` separator.  It is not possible to write the
+    trace output to stdout when specifying ``--enable-mpi``, this will result in
+    an error.
 
---enable-mpi .. _enablempi option:
+--enable-mpi  .. _enablempi option:
```

Comment:
Name suggestion: --gather-reports (or something similar)

My concern is that the name `--enable-mpi` is not very intuitive, as you don't know right away for what MPI is being enabled (the actual functionality or feature you're enabling).

Or is it that it's important to make it clear how report gathering is being implemented? If so, perhaps a name like `--mpi-report-gather` could work.

Or is the idea there will be multiple MPI-related features and you want this option to enable all of them?